packages/fluentui/react-northstar/src/themes/teams/components/RadioGroup/radioGroupItemStyles.ts

```diff
@@ -63,23 +63,23 @@ export const radioGroupItemStyles: ComponentSlotStylesPrepared<RadioGroupItemSty
 
   indicator: ({ props: p, variables: v }): ICSSInJSStyle => ({
     margin: `0 ${pxToRem(12)} 0 0`,
-    borderRadius: '50%',
-    borderWidth: '1px',
-    borderStyle: 'solid',
-    borderColor: v.indicatorBorderColorDefault,
+    outline: 0,
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
     width: pxToRem(12),
     height: pxToRem(12),
+    verticalAlign: 'midddle',
+    color: v.indicatorBorderColorDefault,
 
     ...(p.checked && {
-      background: v.indicatorBackgroundColorChecked,
+      color: v.indicatorBackgroundColorChecked,
       borderColor: 'transparent',
     }),
 
     ...(p.disabled && {
       borderColor: v.colorDisabled,
```

Comment:
I don't think we need the borderColor props anymore

Reply:
Removed